fs-storage/src/file_storage.rs

```diff
@@ -0,0 +1,246 @@
+use std::fmt::Debug;
+use std::fs::{self, File};
+use std::io::{BufRead, BufReader, BufWriter, Write};
+use std::str::FromStr;
+use std::time::SystemTime;
+use std::{
+    collections::HashMap,
+    fmt::Display,
+    path::{Path, PathBuf},
+};
+
+use data_error::{ArklibError, Result};
+
+const LOG_PREFIX: &str = "[file-storage]";
```

Comment:
This came from Kotlin implementation, but let's remove it here. Rust logs stuff a bit differently.